### PR TITLE
Update canvas.rst

### PR DESCRIPTION
**Pull request by ZviBaratz**
Minor typo and styling adjustments.

### CODE REVIEW

File: docs/userguide/canvas.rst [hunk 1]
```diff
@@ -959,11 +959,11 @@ Map & Starmap
 -------------
 
 :class:`~celery.map` and :class:`~celery.starmap` are built-in tasks
-that calls the task for every element in a sequence.
+that call the calling task for every element in a sequence.
```
Comment (thedrow) [inline docs/userguide/canvas.rst:962]: Why did you change the phrasing here?

Comment (ZviBaratz) [reply docs/userguide/canvas.rst:962]: "calls" is grammatically incorrect and therefore was changed to "call". 
I added "calling" because I thought it creates a clearer distinction between the "built-in tasks" (i.e. `map` and `starmap`) at the beginning of the sentence, and the task that in executed over the sequence. It's just a proposition.

Comment (thedrow) [inline docs/userguide/canvas.rst:962]:
```suggestion
that call the provided calling task for every element in a sequence.
```

How's that? Clearer?

Comment (ZviBaratz) [reply docs/userguide/canvas.rst:962]: Absolutely :+1: :dart:

**Review by thedrow — APPROVED** [review]
